exasol_integration_test_docker_environment/lib/base/base_task.py

```diff
@@ -153,7 +153,7 @@ def get_output_path(self) -> Path:
 
     def _get_output_path_for_job(self) -> Path:
         return Path(build_config().output_directory,
-                    "jobs", self.job_id)
+                    "jobs", self.job_id) # type: ignore
```

Comment:
What is the problem here?

Reply:
Same as above. As I fixed the job_id to 
```
job_id : str = luigi.Parameter() # type: ignore
```
AFTER fixing this code location, there is no need more to ignore type check.
Removed it